application/client-test/test/oasis-test.js

```diff
@@ -0,0 +1,52 @@
+const assert = require('assert');
+const config = require('../../config');
+const oasis = require('@oasislabs/client');
+const process = require('process');
+
+let bytecode = require('fs').readFileSync(config.WASM);
+
+const chains = {
+  local: {
+    gateway: new oasis.gateways.Web3Gateway(
+      config.WEB3_GATEWAY_URL,
+      oasis.Wallet.fromMnemonic(config.MNEMONIC)
+    ),
+    completion: chain => chain.gateway.disconnect(),
```

Comment:
You no longer need `completion` as all oasis gateways now have a `disconnect()` method.